packages/backend/src/config/logger.ts

```diff
@@ -0,0 +1,44 @@
+import { createLogger, format, transports } from 'winston'
+import DailyRotateFile from 'winston-daily-rotate-file'
+import path from 'path'
+import { existsSync, mkdirSync } from 'fs'
+
+const env = process.env.NODE_ENV || 'development'
```

Comment:
With the changes mentioned above we should be able to do:

```suggestion
import { existsSync, mkdirSync } from 'fs'
import { env } from './env'
```